salt/states/pkg.py

```diff
@@ -1053,8 +1053,16 @@ def installed(
     if not isinstance(version, six.string_types) and version is not None:
         version = str(version)
 
+    refreshed = False
```

Comment:
Can we use something other than ``refreshed`` here? It's too close to ``refresh`` and will be confusing to maintain.